Example "Stop all running containers" has invalid syntax

File: [engine/api/sdk/examples.md](https://github.com/docker/docs/blob/main/engine/api/sdk/examples.md?plain=1#L352)

In the example **"Stop all running containers"**, the third parameter of function **ContainerStop(ctx, container.ID, nil)** should not be **nil**.
When trying to run the code snippet with `go run main.go` I get this error:
```bash
./main.go:26:50: cannot use nil as container.StopOptions value in argument to cli.ContainerStop
```

--- a/engine/api/sdk/examples.md
+++ b/engine/api/sdk/examples.md
@@ -331,6 +331,7 @@ import (
 	"fmt"
 
 	"github.com/docker/docker/api/types"
+	containertypes "github.com/docker/docker/api/types/container"
 	"github.com/docker/docker/client"
 )
 
@@ -349,7 +350,8 @@ func main() {
 
 	for _, container := range containers {
 		fmt.Print("Stopping container ", container.ID[:10], "... ")
-		if err := cli.ContainerStop(ctx, container.ID, nil); err != nil {
+		noWaitTimeout := 0 // to not wait for the container to exit gracefully
+		if err := cli.ContainerStop(ctx, container.ID, containertypes.StopOptions{Timeout: &noWaitTimeout}); err != nil {
 			panic(err)
 		}
 		fmt.Println("Success")

